### PR TITLE
added Troubleshooting section in README.

### PR DESCRIPTION
**Pull request by droot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,10 @@ Provide clean library abstractions with clear and well exampled godocs.
 - Provide code generators to maintain common boilerplate that can't be addressed by interfaces
   - Driven off of `//+` comments
 - Provide bootstrapping commands to initialize new packages
+
+### Troubleshooting
+
+If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system.](https://github.com/kubernetes-sigs/kubebuilder/issues)
```
Comment (Liujingfang1) [inline README.md]: We can add a subtitle `bugs and feature requests` here.

File: README.md [hunk 2]
```diff
@@ -67,3 +67,10 @@ Provide clean library abstractions with clear and well exampled godocs.
...
+Before you file an issue, please search existing issues to see if your issue is already covered.
+
+For realtime discussion,  you can join the [#kubebuilder](https://slack.k8s.io/#kubebuilder) slack channel. Slack requires registration, but the Kubernetes team is open invitation to anyone to register here. Feel free to come and ask any questions.
```
Comment (Liujingfang1) [inline README.md]: We can add a subtitile `slack` here.

File: README.md (hunk 1 above)
Comment (droot) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (droot) [reply README.md]: Done

**Comment by pmorie** [comment]
lgtm

**Pull request merged**